x/stake/keeper/delegation.go

```diff
@@ -600,6 +600,9 @@ func (k Keeper) BeginRedelegation(ctx sdk.Context, delAddr sdk.AccAddress,
 	}
 
 	rounded := returnAmount.TruncateInt()
+	if rounded.IsZero() { //TODO design consideration
+		return types.Redelegation{}, types.ErrVerySmallRedelegation(k.Codespace())
+	}
```

Comment:
The real doozy of a bug fix

Reply:
Indeed - what is the design consideration which is "TODO" here? Could we link to an issue if you think there may be further follow-up needed?

Reply:
Agreed further elaboration is needed here. Idk what design consideration is needed.

Reply:
reflecting, I don't think follow up is needed - removing comment